heareval/evaluation/task_evaluation.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+"""
+Contains a set of various evaluation scores. Runs evaluation on the test predictions
+for predictions on a HEAR task.
+Would it make sense to move all the score functions over to a file called scores?
+
+TODO: Fix AUC
```

Comment:
Is AUC still broken?

Reply:
AUC doesn't really make sense for multiclass in the way we thought, unless we want macroaveraging

Reply:
Removed